packages/vaadin-combo-box/src/vaadin-combo-box-light.d.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+export * from '@vaadin/combo-box/src/vaadin-combo-box-light.js';
```

Comment:
Please move legacy `ComboBoxLightElement` aliases to this package and remove them from `combo-box`.

Reply:
Done.